### PR TITLE
Circle CI 2.0 support

### PR DESCRIPTION
**Pull request by damianlegawiec**

### CODE REVIEW

**Comment by kushniryb** [comment]
Looks good to me 👍

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+      # Which version of bundler?
+      - run:
+          name: Which bundler?
+          command: bundle -v
+
+      # Restore bundle cache
+      - restore_cache:
+          keys:
+            - spree-bundle-v2-{{ .Branch }}
```
Comment (bbonislawski) [inline .circleci/config.yml]: Why `v2` ?

Comment (damianlegawiec) [reply .circleci/config.yml]: @bbonislawski v2 == circle 2.0 :)

Comment (bbonislawski) [reply .circleci/config.yml]: Let's say it makes sense. Everything else is 👍

**Pull request merged**